dockerfiles/azure-repos/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM node:12-slim
+
+LABEL maintainer="Snyk Ltd"
+
+RUN npm install --global snyk-broker
+
+RUN apt-get update && apt-get install -y ca-certificates
+
+# Don't run as root
+WORKDIR /home/node
+USER node
+
+# Generate default accept filter
+RUN broker init azure-repos
+
+######################################
+# Custom Broker Client configuration #
+# Redefine in derived Dockerfile,    #
+# or provide as runtime args `-e`    #
+######################################
+
+# https://dev.azure.com/{azure-repos-org}
+ENV AZURE_REPOS_ORG <azure-repos-org>
+# Guide how to get/create the token https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=preview-page
+# Scopes: Ensure Custom defined is selected and under Code select Read & write
+ENV AZURE_REPOS_TOKEN <azure-repos-token>
+# NOTA BENE: without `/` character in the end of URL
+ENV AZURE_REPOS_HOST dev.azure.com
+
+# Your unique Broker identifier
+ENV BROKER_TOKEN <broker-token>
+ENV BROKER_CLIENT_VALIDATION_BASIC_AUTH "PAT:$AZURE_REPOS_TOKEN"
+ENV BROKER_SERVER_URL https://broker.snyk.io
```

Comment:
this one and the next one should not be here - those are defaults in .env file, and are not custom-based. You can move them up, before line 16, or remove them

Reply:
You are right. No need to duplicate this values here